defaults/main.yml

```diff
@@ -0,0 +1,40 @@
+---
+synapse_user: synapse
+synapse_install_dir: /opt/synapse
+synapse_data_dir: /etc/synapse
```

Comment:
`/etc` is for configuration files, see https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard#Directory_structure